src/software/thunderscope/robot_diagnostics/drive_and_dribbler_widget.py

```diff
@@ -0,0 +1,206 @@
+from PyQt6.QtCore import Qt
+from PyQt6.QtWidgets import *
+
+
+class DriveAndDribblerWidget(QWidget):
+    def __init__(self):
+        QWidget.__init__(self)
+        layout = QVBoxLayout()
+
+        # Initialize tab screen
+        tabs = QTabWidget()
+        per_wheel_tab = QWidget()
+        direct_velocity_tab = QWidget()
+        tabs.setStyleSheet("background-color: black; color:white")
+        # Add tabs
+        tabs.addTab(per_wheel_tab, "Direct Per-Wheel Control")
+        tabs.addTab(direct_velocity_tab, "Direct Velocity Control")
+
+        # Create first tab
+        per_wheel_tab.grid = QVBoxLayout()
+        per_wheel_tab.grid.addWidget(self.setUpDirectPerWheel("Drive"))
+        per_wheel_tab.grid.addStretch(1)
+        per_wheel_tab.setLayout(per_wheel_tab.grid)
+
+        # Create second tab
+        direct_velocity_tab.grid2 = QVBoxLayout()
+        direct_velocity_tab.grid2.addWidget(self.setUpDirectVelocity("Drive"))
+        direct_velocity_tab.grid2.addStretch(1)
+        direct_velocity_tab.setLayout(direct_velocity_tab.grid2)
```

Comment:
i think the `setUpDirectVelocity` needs to be renamed to `setup_up_direct_velocity`